arraycontext/impl/pyopencl/fake_numpy.py

```diff
@@ -134,11 +136,13 @@ def min(self, a, axis=None):
 
         def _rec_min(ary):
             if axis not in [None, tuple(range(ary.ndim))]:
-                raise NotImplementedError(f"Min. over '{axis}' axes not supported.")
+                raise NotImplementedError(f"min over '{axis}' axes not supported.")
+
             return cl_array.min(ary, queue=queue)
 
         result = rec_map_reduce_array_container(
-                partial(reduce, partial(cl_array.minimum, queue=queue)),
+                lambda arys: reduce(
+                    partial(cl_array.minimum, queue=queue), arys, np.inf),
```

Comment:
Actually, I've removed these changes because they're a bit complicated and distinct.

Not sure if it's worth bothering until there's an actual problem? I can make a separate PR with them though.